### PR TITLE
New package: cli53 (AWS Route53 admin CLI tool)

### PR DESCRIPTION
**Pull request by benley**

### CODE REVIEW

File: pkgs/tools/admin/cli53/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, pythonPackages, fetchurl }:
+
+buildPythonPackage rec {
+  name = "cli53-${version}";
+  namePrefix = "";
+  version = "0.4.4";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/c/cli53/${name}.tar.gz";
+    md5 = "da4ba52e36a53aadaa537397e735fbfd";
```
Comment (spwhitt) [inline pkgs/tools/admin/cli53/default.nix]: md5 is deprecated. We prefer sha256.

File: pkgs/tools/admin/cli53/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, pythonPackages, fetchurl }:
+
+buildPythonPackage rec {
+  name = "cli53-${version}";
+  namePrefix = "";
```
Comment (spwhitt) [inline pkgs/tools/admin/cli53/default.nix]: This seems unused, you can delete it.

Comment (benley) [inline pkgs/tools/admin/cli53/default.nix]: `namePrefix = "";` is necessary to stop buildPythonPackage from prefixing the package's name with `python27-`.  It's a behaviour that makes sense for libraries, but not so much for standalone apps meant for direct installation.

File: pkgs/tools/admin/cli53/default.nix (hunk 1 above)
Comment (benley) [inline pkgs/tools/admin/cli53/default.nix]: oops, I overlooked that. Fixing.

**Comment by benley** [comment]
fyi the new build failure is unrelated to this change.

**Comment by benley** [comment]
Ping - is this okay now? Any objections?

**Comment by spwhitt** [comment]
LGTM

**Comment by benley** [comment]
can someone kindly merge this?

**Comment by benley** [comment]
thanks :-)

**Pull request merged**